src/python/foglamp/device/__main__.py

```diff
@@ -16,25 +18,19 @@
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
+_logger = logger.setup("Device", level=20)
 
-# TODO: Support --help, --name plugin, etc.
-try:
-    plugin = sys.argv[1]
-    index = plugin.find('--name=')
-    if index == -1:
-        plugin = None
-    else:
-        plugin = plugin
-except IndexError:
-    plugin = None
 
-if plugin is None:
-    # TODO: delete this
-    plugin = 'CoAP'
+plugin = Parser.get('--name')
```

Comment:
The plugin is not passed in the name parameter the plugin comes form the configuration. The name is used to work out the name of the configuration section to look at https://docs.google.com/document/d/1c1dmMxH36IQNA_gFwxTPttyCeEStsrSxWYUqKeBbfNk/edit#heading=h.o2dnspmkfx4p